deployment/windows/build-jellyfin.ps1

```diff
@@ -131,6 +132,23 @@ function Cleanup-NSIS {
     Remove-Item "$tempdir/nsis/" -Recurse -Force -ErrorAction Continue | Write-Verbose
     Remove-Item "$tempdir/nsis.zip" -Force -ErrorAction Continue | Write-Verbose
 }
+
+function Install-TrayApp {
+    param(
+        [string]$ResolvedInstallLocation,
+        [string]$Architecture
+    )
+    Write-Verbose "Checking Architecture"
+    if($Architecture -ne 'x64'){
+        Write-Warning "No builds available for your selected architecture of $Architecture"
+        Write-Warning "The tray app will not be available."
+    }else{
+        Write-Verbose "Downloading Tray App and copying to Jellyfin location"
+        [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+        Invoke-WebRequest -Uri https://github.com/jellyfin/jellyfin-windows-tray/releases/download/v1.0/JellyfinTray.exe -UseBasicParsing -OutFile "$installLocation/JellyfinTray.exe" | Write-Verbose
```

Comment:
We will need to do something similar to this:

https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8

or host it on the repo server.

Reply:
I suppose I could flip it around and get a version like NSSM, but I also don’t expect the tray app to change until .NET Core 3.0 is in master, and then I can switch this to a framework dependent exe.